builders/interoperability/xcm/xc20/send-xc20s/xtokens-pallet.md

```diff
@@ -7,7 +7,7 @@ description: This guide provides an introduction to the X-Tokens Pallet and expl
 
 ## Introduction {: #introduction }
 
-Building an XCM message for fungible asset transfers is not an easy task. Consequently, there are wrapper functions and pallets that developers can leverage to use XCM features on Polkadot and Kusama. One example of such wrappers is the [X-Tokens](https://github.com/moonbeam-foundation/open-runtime-module-library/tree/master/xtokens){target=\_blank} Pallet, which provides different methods to transfer fungible assets via XCM.
+Building an XCM message for fungible asset transfers is not an easy task. Consequently, there are wrapper functions and pallets that developers can leverage to use XCM features on Polkadot and Kusama. One example of such wrappers is the [X-Tokens](https://github.com/moonbeam-foundation/open-runtime-module-library/blob/master/xtokens){target=\_blank} Pallet, which provides different methods to transfer fungible assets via XCM.
```

Comment:
When I go to the blob link it redirects to the tree link, i.e. the old link.

Reply:
good catch! thanks!